packages/react/src/BackboneElementDisplay/BackboneElementDisplay.tsx

```diff
@@ -1,25 +1,56 @@
-import { getPathDisplayName, tryGetDataType, TypedValue } from '@medplum/core';
+import {
+  buildElementsContext,
+  ElementsContextType,
+  getPathDisplayName,
+  isEmpty,
+  tryGetDataType,
+  TypedValue,
+} from '@medplum/core';
 import { DEFAULT_IGNORED_NON_NESTED_PROPERTIES, DEFAULT_IGNORED_PROPERTIES } from '../constants';
 import { DescriptionList, DescriptionListEntry } from '../DescriptionList/DescriptionList';
 import { ResourcePropertyDisplay } from '../ResourcePropertyDisplay/ResourcePropertyDisplay';
 import { getValueAndType } from '../ResourcePropertyDisplay/ResourcePropertyDisplay.utils';
+import { useContext, useMemo } from 'react';
+import { ElementsContext } from '../ElementsInput/ElementsInput.utils';
+import { maybeWrapWithContext } from '../utils/maybeWrapWithContext';
+
+const EXTENSION_KEYS = new Set(['extension', 'modifierExtension']);
+const IGNORED_PROPERTIES = new Set(DEFAULT_IGNORED_PROPERTIES.filter((prop) => !EXTENSION_KEYS.has(prop)));
```

Comment:
Suuuper minor, but we microbenchmarked this elsewhere and found that `array.includes()` was faster than `set.has()` for sets under 20 elements.